lib/cli.js

```diff
@@ -139,8 +106,15 @@ function run () {
       cli.info("Runtime Environment (" + releaseName + " " + process.version + " on " + process.platform + ")");
     } else {
       cliArgs.forEach(function (file) {
-        var data = convertFile(file, options);
-        outputResult(data, file, cliOptions);
+        cli.debug('converting file ' + file);
+        if (options.timings) {
+          var timings = asciidoctor.Timings.$new();
+          var instanceOptions = Object.assign({}, options, { timings: timings })
+          asciidoctor.convertFile(file, instanceOptions);
+          timings.$print_report(Opal.gvars.stderr, file);
```

Comment:
It should be `stdout` no ? By default the `print_report` is using `stdout` in core: https://github.com/asciidoctor/asciidoctor/blob/25ca4dafc34d733f5f9c1e723038f93386c318ba/lib/asciidoctor/timings.rb#L50

Reply:
The CLI overrides this. See https://github.com/asciidoctor/asciidoctor/blob/25ca4dafc34d733f5f9c1e723038f93386c318ba/lib/asciidoctor/cli/invoker.rb#L124

Otherwise, the output gets mixed with the document piped to stdout.